rest-api/tools/consent_upload_validation.py

```diff
@@ -0,0 +1,139 @@
+""" read from csv (assumes participant id column header is [pmi_id] and paired site header is
+    [paired_site]).
+     Checks PTC-UPLOADS-ALL-OF-US-RDR-PROD bucket and the bucket given in args (i.e. aouXXX)
```

Comment:
grammar nit: I think this should be e.g. rather than i.e.

Reply:
done.